### PR TITLE
new for ESap046

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: new/PRS12940Danel.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዳንኤል፡<addName type="epithet"> ፈላሲ፡</addName></persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dānʾel  <addName type="epithet">falāsi</addName></persName>
+                    <persName xml:lang="en" corresp="#n1"> Daniel the Stranger</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">According to the embedded colophon in <ref type="work" corresp="LIT1491Gadlas"/>, he was the one who has “translated” (za-ʾaʿlawkǝwwā) the work (Gadla
+                        and Taʾāmra samāʿtāta Ṗarāqliṭos</occupation>
```
Comment (thea-m) [inline new/PRS12940Danel.xml:53]: Taʾammǝra

File: new/PRS12940Danel.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12940Danel" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title> Dānʾel “the falāsi”</title>
```
Comment (thea-m) [inline new/PRS12940Danel.xml:8]: we usually have Dānǝʾel, which is also used in the traces annotations.

File: new/PRS12939TasfaSeyonScribe.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተስፋ፡ ጽዮን፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Tasfā Ṣǝyon</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <birth/>
+                    <death/>
```
Comment (thea-m) [inline new/PRS12939TasfaSeyonScribe.xml:53]: I always prefer to delete these empty elements to clean up the records.

Comment (eu-genia) [reply new/PRS12939TasfaSeyonScribe.xml]: Ok all done

**Pull request merged**